# Add Message header
Header header

# Data is here in DOT format
string sg_data

# For each object, the four important components are
#float64    object_distance
#float64    object_direction
#float64    object_speed 
#float64    object_orientation 
